lambda/variables.tf

```diff
@@ -1,4 +1,5 @@
 variable "name" {}
+variable "lambda_layer_name" { default = "lambda_layer_rds_ca" }
```

Comment:
I think the variable name should imply that it is a CA lambda, for example: lambda_rds_ca_layer_name